From 593e8cc0b374aa7b20d3d961c57feb9bab508979 Mon Sep 17 00:00:00 2001
From: =?utf8?q?Bernhard=20Rosenkr=C3=A4nzer?=
 <bernhard.rosenkranzer@linaro.org>
Date: Sun, 30 Dec 2012 12:18:00 -0800
Subject: [PATCH] Fix build on ARM with binutils >= 2.23.51.0.6 GAS doesn't
 seem to like spaces in vld1 anymore, so remove those.

---
 common/arm/pixel-a.S |   16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/common/arm/pixel-a.S b/common/arm/pixel-a.S
index 8bce3b6..fa0004f 100644
--- a/common/arm/pixel-a.S
+++ b/common/arm/pixel-a.S
@@ -68,45 +68,45 @@ SAD4_ARMV6 8
 
 
 .macro SAD_START_4 align:vararg
-    vld1.32     {d1[]}, [r2 \align], r3
+    vld1.32     {d1[]}, [r2\align], r3
     vld1.32     {d0[]}, [r0,:32], r1
     vabdl.u8    q8,  d0,  d1
 .endm
 
 .macro SAD_4 align:vararg
-    vld1.32     {d1[]}, [r2 \align], r3
+    vld1.32     {d1[]}, [r2\align], r3
     vld1.32     {d0[]}, [r0,:32], r1
     vabal.u8    q8,  d0,  d1
 .endm
 
 .macro SAD_START_8 align:vararg
-    vld1.64     {d1}, [r2 \align], r3
+    vld1.64     {d1}, [r2\align], r3
     vld1.64     {d0}, [r0,:64], r1
     vabdl.u8    q8,  d0,  d1
 .endm
 
 .macro SAD_8 align:vararg
-    vld1.64     {d1}, [r2 \align], r3
+    vld1.64     {d1}, [r2\align], r3
     vld1.64     {d0}, [r0,:64], r1
     vabal.u8    q8,  d0,  d1
 .endm
 
 .macro SAD_START_16 align:vararg
-    vld1.64     {d2-d3}, [r2 \align], r3
+    vld1.64     {d2-d3}, [r2\align], r3
     vld1.64     {d0-d1}, [r0,:128], r1
     vabdl.u8    q8,  d0,  d2
-    vld1.64     {d6-d7}, [r2 \align], r3
+    vld1.64     {d6-d7}, [r2\align], r3
     vabdl.u8    q9,  d1,  d3
     vld1.64     {d4-d5}, [r0,:128], r1
 .endm
 
 .macro SAD_16 align:vararg
     vabal.u8    q8,  d4,  d6
-    vld1.64     {d2-d3}, [r2 \align], r3
+    vld1.64     {d2-d3}, [r2\align], r3
     vabal.u8    q9,  d5,  d7
     vld1.64     {d0-d1}, [r0,:128], r1
     vabal.u8    q8,  d0,  d2
-    vld1.64     {d6-d7}, [r2 \align], r3
+    vld1.64     {d6-d7}, [r2\align], r3
     vabal.u8    q9,  d1,  d3
     vld1.64     {d4-d5}, [r0,:128], r1
 .endm
-- 
1.7.10.4

